client/blocks/subscriptions-module-banner/use-subscription-banner.ts

```diff
@@ -4,14 +4,20 @@ import isActivatingJetpackModule from 'calypso/state/selectors/is-activating-jet
 import isJetpackModuleActive from 'calypso/state/selectors/is-jetpack-module-active';
 import { isJetpackSite } from 'calypso/state/sites/selectors';
 
-export default function useSubscriptionBanner( siteId: number, isDismissed?: boolean ) {
+export default function useSubscriptionBanner( siteId?: number, isDismissed?: boolean ) {
```

Comment:
Note to self: `siteId` becomes optional just in case the site selector hasn't loaded or is in the middle of load the site.